openreview/venue/webfield/paperBidWebfield.js

```diff
@@ -1,19 +1,19 @@
 // Webfield component
-const COMMITTEE_NAME = ''
 const BID_INSTRUCTIONS = ''
+const commitee_name = entity.content.commitee_name.value
 
 return {
   component: 'BidConsole',
   version: 1,
   properties: {
     header: {
-      "title": `${COMMITTEE_NAME} Bidding Console`,
+      "title": `${commitee_name.endsWith('s') ? commitee_name.slice(0, -1) : commitee_name} Bidding Console`,
       "instructions": BID_INSTRUCTIONS
```

Comment:
I think the bid instructions should be included in this template and use varibles from the domain content. I don't think it needs to be set by python.

Reply:
I will add this to another PR